docker/zfaucet/entrypoint.sh

```diff
@@ -5,6 +5,7 @@ set -e
 python manage.py makemigrations
 python manage.py migrate
 python manage.py collectstatic --no-input
-python manage.py check_zcashd --wait 20
+python manage.py check_zcashd --wait 120
```

Comment:
This is a better interval, especially when block dimes decrease with blossum activation